utils_cv/detection/plot.py

```diff
@@ -65,26 +66,46 @@ def plot_boxes(
         draw = ImageDraw.Draw(im)
         font = get_font(size=plot_settings.text_size)
 
+        # get color map and convert to list of integer tuples
+        if not (plot_settings.rect_color and plot_settings.text_color):
```

Comment:
this case should never happen since PlotSettings is initialized with a `rect_color` and a `text_color`

Reply:
i implemented it such that if plot_settings.rect_color is set to None, it instead cycles through all colors.